app/models/dto.py

```
@@ -63,6 +63,42 @@ class AvailabilityRequest(BaseModel):
+            raise ValueError(f"날짜 형식이 올바르지 않습니다. (YYYY-MM-DD 포맷 필요, 입력값: {self.date})")
+        try:
+            req_date = datetime.strptime(self.date, "%Y-%m-%d").date()
+        except ValueError as err:
+            raise ValueError(f"날짜 형식이 올바르지 않습니다. (YYYY-MM-DD 포맷 필요, 입력값: {self.date})") from err
+        
+        kst = timezone(timedelta(hours=9))
+        now_kst = datetime.now(kst)
+        today = now_kst.date()
+        
+        if req_date < today:
+            raise ValueError(f"과거 날짜({self.date})는 예약할 수 없습니다.")
```

Ruff (0.15.1)

[warning] Avoid specifying long messages outside the exception class

(TRY003)